spec/PeriodicBackgroundSync-index.bs

```diff
@@ -92,3 +94,95 @@ Fetch requests within the <code>periodicsync</code> event while [=in the backgro
 
 ## History Leaking ## {#history-leaking}
 Fetch requests within the <code>periodicsync</code> event while [=in the background=] may reveal something about the client's navigation history to middleboxes on networks different from the one used to create the [=periodicsync registration=]. For instance, the client might visit site https://example.com, which registers a <code>periodicsync</code> event, but based on the implementation, might not fire until after the user has navigated away from the page and changed networks. Middleboxes on the new network may see the fetch requests that the <code>periodicsync</code> event makes. The fetch requests are HTTPS so the request contents will not be leaked but the domain may be (via DNS lookups and IP address of the request). To prevent this leakage of browsing history, the user agent MAY choose to only fire <code>periodicsync</code> events on the network the [=periodicsync registration=] was made on, with the understanding that it will reduce usability by not allowing synchronization opportunistically.
+
+Algorithms {#algorithms}
+=========================
+## <dfn>Calculate a time to fire</dfn> ## {#caculate-time-to-fire}
+This section describes how a user agent can calculate the [=time to fire=] for a [=periodicsync registration=], <var>registration</var>. The time interval between periodicSync events for |registration| MUST be greater than or equal to the [=minInterval=] value associated with |registration|.
```

Comment:
Re: defining an options, minInterval should be scoped by that.

Reply:
Done. In a previous CL, you had suggested getting rid of options, and I was thus avoiding referring to it.

Reply:
This is way I was suggesting getting rid of it :)
It's an infra layer of abstraction that doesn't add much value except for the resulting boilerplate when referencing it

Reply:
There's a good reason to keeping the options dictionary -- it allows us to extend the API with more options in the future without breaking existing usage. There's also a line explaining this when I introduce options.